lib/src/io/tar_file_encoder.dart

```diff
@@ -17,8 +17,8 @@ class TarFileEncoder {
   void tarDirectory(Directory dir,
       {int compression = STORE, String? filename}) {
     final dirPath = dir.path;
-    var tar_path = filename ?? '${dirPath}.tar';
-    final tgz_path = filename ?? '${dirPath}.tar.gz';
+    var tar_path = filename ?? '$dirPath.tar';
```

Comment:
Just curious if not using explicit ${var} is a personal preference or has some impact. My preference is with the { }s.

Reply:
Sorry its because of the Android Studio IDE suggesting and bothering eyes, I personally have no preference, just it is rather distracting to have yellow underlines.

Reply:
Then that's good. I haven't used Android Studio, but editor "suggestions" annoy me as well :-). Dart has been pretty awful about changing the recommended coding style guide over the years, I can't keep up with it anymore.